dso-l1/src/main/java/com/tc/object/EntityClientEndpointImpl.java

```diff
@@ -50,18 +52,35 @@
   /**
    * @param entityDescriptor The server-side entity and corresponding client-side instance ID.
    * @param invocationHandler Called to handle "invokeAction" requests made on this end-point.
+   * @param fetchID
```

Comment:
It looks like this was removed.  The JavaDoc should probably be updated, anyway.

There is other JavaDoc which is rendered stale by this change but I don't want to block this based on those.